src/main/java/org/spectra/cluster/model/cluster/ICluster.java

```diff
@@ -34,7 +35,7 @@
      *
      * @return The charge
      */
-    int getPrecursorCharge();
+    Integer getPrecursorCharge();
```

Comment:
@ypriverol Same question as above: why the `Integer`? `int` is quite a bit smaller